### ISSUE TITLE
Move this repository into backdrop-contrib (rather than fork into it)

### ISSUE THREAD

**Issue by quicksketch** [Lowell20/profile_picture_plus#1]
Hi @Lowell20! It looks like instead of this repository being *moved* into the backdrop-contrib group, it's been forked into it. I hadn't realized that would even be possible. Could you delete the repository at https://github.com/backdrop-contrib/profile_picture_plus/settings and then *move* this repository into backdrop-contrib at https://github.com/Lowell20/profile_picture_plus/settings? That will put the canonical version in backdrop-contrib instead of it being a fork Otherwise when other people fork your project to submit pull requests, they'll fork from this copy instead of the one in backdrop-contrib.

**Comment by Lowell20** [comment]
I tried, couldn't find the move process. Thanks for noticing, I wanted to
ask about that.

If I delete my copy, would that do the trick? I might as well try to find
out.

Lowell Johnson
715-579-9276

**Comment by Lowell20** [comment]
I am having an issue transferring the repo. I did find it in the settings, thanks. I am trying to transfer it to 
backdrop-contrib but it doesn't seem to accept that name. I guess I am missing something simple. Will keep looking.

**Comment by Lowell20** [comment]
Looks like this might be a reason. Do I have admin rights at the receiving repo?
https://help.github.com/articles/transferring-a-repository/#transferring-from-a-user-to-an-organization

**Comment by Lowell20** [comment]
I decided to create a new empty repo at backdrop-contrib and then push into it from local.
Should I now delete this repo?

**Comment by quicksketch** [comment]
> Looks like this might be a reason. Do I have admin rights at the receiving repo?

Yep, everyone has admin rights. It the only permission that allows creating repositories.

> I decided to create a new empty repo at backdrop-contrib and then push into it from local.

That works too. :smile:

>  Should I now delete this repo?

Yes, it might help reduce confusion for people searching for the module. Thanks again!
